channels/ir.m3u

```diff
@@ -107,6 +107,8 @@ https://live.kalemehtv.tv/live/ngrp:kalemeh_all/playlist.m3u8
 https://stream.alkafeel.net/live/alkafeel/rAa5PGot1/manifest.m3u8
 #EXTINF:-1 tvg-id="KarbalaLive4.ir" tvg-name="Karbala Live 4" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala Live 4 (360p)
 https://stream.alkafeel.net/live/alkafeel/rAa5PGot4/manifest.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Karbala" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala TV
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/c/karbalaatv/live
```

Comment:
Karbala Satellite Channel is Iraqi channel.

https://www.youtube.com/c/karbalaatv/about
https://www.lyngsat.com/tvchannels/iq/Karbala-Satellite-Channel.html

Reply:
Thanks fixed